test/document-test.js

```diff
@@ -149,29 +151,29 @@ describe("Document", () => {
 
     it("can set cookie", () => {
       document.cookie = "_new=2";
-      expect(document.cookie).to.equal("_ga=1;_new=2;");
+      expect(document.cookie).to.equal("_ga=1;_new=2");
     });
 
     it("overwrites cookie with same name", () => {
       document.cookie = "_writable=2";
       document.cookie = "_writable=3";
-      expect(document.cookie).to.equal("_ga=1;_writable=3;");
+      expect(document.cookie).to.equal("_ga=1;_writable=3");
     });
 
     it("URI encodes when setting value", () => {
       document.cookie = "_writable=2 3";
-      expect(document.cookie).to.equal("_ga=1;_writable=2%203;");
+      expect(document.cookie).to.equal("_ga=1;_writable=2 3");
```

Comment:
Test says URI encodes, but it does not encode...

Reply:
A browser does not. So I guess the test was wrong